app/models/decorators/mvi_user_decorator.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
-class Decorators::MviUserDecorator
-  MVI_SERVICE = VetsAPI::Application.config.mvi_service
+require 'mvi/service_factory'
 
+class Decorators::MviUserDecorator
   def initialize(user)
     @user = user
+    @mvi_service = MVI::ServiceFactory.get_service(mock_service: ENV['MOCK_MVI_SERVICE'])
```

Comment:
Do you want to maybe make it be:

```ruby
@mvi_service = MVI::ServiceFactory.get_service(mock_service: ENV['MOCK_MVI_SERVICE'] || true)
```

or would that make things confusing? The reason I ask is, this will likely break the build and shouldn't be merged until the devops ticket gets merged first.